glossary.html

```diff
@@ -1,9 +1,39 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-    <title></title>
+    <title>YangZhou</title>
+    <meta charset="utf-8">
 </head>
 <body>
-
+    <header>
```

Comment:
This file had no content in it, and your links.html file had the Acronym list that should have  been in here, so I moved it to here.